$NetBSD$

Add placeholder for HAVE_KRB5_COM_ERR_H in config.h.in.

--- include/config.h.in.orig	2005-03-22 14:30:49.000000000 +0100
+++ include/config.h.in	2005-03-22 14:31:34.000000000 +0100
@@ -196,6 +196,9 @@
 /* Define to 1 if you have the <et/com_err.h> header file. */
 #undef HAVE_ET_COM_ERR_H
 
+/* Define to 1 if you have the <krb5/com_err.h> header file. */
+#undef HAVE_KRB5_COM_ERR_H
+
 /* Define if you have the function `fchown'. */
 #undef HAVE_FCHOWN
 
